### PR TITLE
Add method for returns the map with content of fields that the new row was filled

### PR DESCRIPTION
**Pull request by gpbonillas**
-  Change the dependency version from 3.1.6 to 3.1.7 on pom file
-  Add public method for getting the field attribute that has the row content

### CODE REVIEW

**Review by flotho — COMMENTED** [review]
Hi, thanks for your proposal,
any chance to use the same pattern for indentation so that it won't turn the review process into a nightmare for later contributions and modifications?

File: src/main/java/com/odoojava/api/Row.java [hunk 1]
```diff
@@ -250,6 +250,14 @@ public FieldCollection getChangedFields() {
 		return changedFields;
 	}
 
+	/**
+	 * Returns the map with fields content that the row was filled
+	 * @return
+	 */
+	public HashMap<String, Object> getFieldsOpenERPResult() {
```
Comment (flotho) [inline src/main/java/com/odoojava/api/Row.java:257]: Please change the name of your methode and replace openrpresult by Odoo even if the variable is openerp.

**Review by flotho — CHANGES_REQUESTED** [review]
thanks for your proposal : 
* some changes are required to be merged please see my comments
* could you also precise what kind of use case you're dealing with this change and propose it on the readme
* could you also squash you commits once done

File: src/main/java/com/odoojava/api/Context.java [hunk 2]
```diff
@@ -1,6 +1,6 @@
 /*
  *   Copyright 2011, 2013-2014 De Bortoli Wines Pty Limited (Australia)
- *   
+ *
```
[the remark this answers is not in the document]
Comment (gpbonillas) [reply src/main/java/com/odoojava/api/Context.java:3]: Yes, I only added spaces for legibility, but there's no problem to use the same source/pattern.

File: src/main/java/com/odoojava/api/Row.java (hunk 1 above)
Comment (gpbonillas) [reply src/main/java/com/odoojava/api/Row.java:257]: I will change it according to your suggestion and push the commit

**Comment by gpbonillas** [comment]
> thanks for your proposal :
> 
> * some changes are required to be merged please see my comments
> * could you also precise what kind of use case you're dealing with this change and propose it on the readme
> * could you also squash you commits once done

-----
About the use case for **getFieldsOdoo** method:

I need to get fields and then save for tracking purposes when I create a new object.

**Comment by gpbonillas** [comment]
> * propose

The README file was updated with the proposal.

**Comment by gpbonillas** [comment]
> Hi @gpbonillas
> 
> I really appreciate your effort.
> I still have a pbm regarding spaces you use for src/main/java/com/odoojava/api/ObjectAdapter.java for example. All the file is given different from source so it's difficult for me to check everything.
> Could you please change your settings so that the space number still the same as the original one in the project?
> 
> Regards

Sorry!. I update the file, then merge. The changes are: https://github.com/odoo-java/odoo-java-api/pull/13/commits/22e9daa0f0419f1fd1e067240dd20ad3c0e6e33a

**Pull request merged**